### PR TITLE
Issue 5554 - Add more tests to security_basic_test suite

### PR DESCRIPTION
**Pull request by bsimonova**
Description:
Add test for TCP_ERROR maxbersize B2 scenario.

Relates: https://github.com/389ds/389-ds-base/issues/5554

Reviewed by: ???

### CODE REVIEW

**Review by mreynolds389 — APPROVED** [review]
LGTM

File: dirsrvtests/tests/suites/logging/security_basic_test.py [hunk 1]
```diff
@@ -258,6 +267,48 @@ def test_tcp_events(topo, setup_test):
...
 
+def test_tcp_events_maxbersize(topo, setup_test, big_file):
+    """Trigger a TCP_ERROR event B2 that should be logged in the security log
+
+    :id: 85e5ac23-4288-4e55-b8c3-1f8f39e95c2b
+    :setup: Standalone Instance
+    :testype: Non-functional
+    :subtype1: Security
+    :subtype2: Penetration
+    :subsystemteam: sst_idm_ds
```
Comment (droideck) [inline dirsrvtests/tests/suites/logging/security_basic_test.py:278]: I think we need to update https://github.com/389ds/389-ds-base/blob/main/dirsrvtests/testimony.yaml as the validate test fails.

**Review by droideck — APPROVED** [review]
Except for the minor issue, looks good!

File: dirsrvtests/tests/suites/logging/security_basic_test.py (hunk 1 above)
Comment (bsimonova) [reply dirsrvtests/tests/suites/logging/security_basic_test.py:278]: Ah, I see, didn't realize the validation.
There is a requirement to mark tests that fall into Security category in Polarion.
However they are still making changes to it.

I updated the testimony file, @vashirov do you think it's ok like this?
Or should I revert and wait for the final conclusion to the security fields?

Thanks!

Comment (vashirov) [reply dirsrvtests/tests/suites/logging/security_basic_test.py:278]: Yeah, it's fine. `testimony.yml` defines which fields can be present and if they are required or not. So if these fields will be unused, it won't fail the check. And once there will be an agreement on which fields are required, we can update this file.

Comment (bsimonova) [reply dirsrvtests/tests/suites/logging/security_basic_test.py:278]: Thanks!

**Pull request merged**